website/settings/local-travis.py

```diff
@@ -106,4 +106,5 @@ class CeleryConfig(defaults.CeleryConfig):
 DOI_FORMAT = '{prefix}/FK2osf.io/{guid}'
 
 SHARE_ENABLED = False
+
```

Comment:
Nitpick, but extra whitespace that wasn't there before in an otherwise unchanged file.